extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureLoadSpec.java

```diff
@@ -31,24 +31,24 @@
 public class AzureLoadSpec implements LoadSpec
 {
   @JsonProperty
-  private final String storageDir;
+  private final String blobPath;
 
   private final AzureDataSegmentPuller puller;
 
   @JsonCreator
   public AzureLoadSpec(
-      @JacksonInject AzureDataSegmentPuller puller,
-      @JsonProperty("storageDir") String storageDir
```

Comment:
The concept of container and blob is distinct, so I think it makes sense to separate those at least.

From my cursory reading of the azure docs, it seems each storage account is tied to a given set of keys, so unless it's common to work across different storage accounts, I think it makes sense to leave the storage account configuration global where the authentication keys are specified.

If working across accounts is a requirement, then we should have three fields, `storageAccount`, `container`, and `blob`